packages/bpk-component-floating-notification/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bpk-component-floating-notification",
+  "version": "1.0.0",
```

Comment:
Can you reset this to `0.0.1` as otherwise our release process won't allow us to publish it as `1.0.0`

Reply:
Sure, no problem